docs/events/winter_jam_2018.md

```diff
@@ -0,0 +1,44 @@
+<p align="center"> 
+<img src="https://cdn.discordapp.com/attachments/463961352432517123/516812016980066307/winter-jam-header-2018.gif">
+</p>
+
+I am happy to announce MMD's newest Jam, Winter Jam! It tastes like snow!  
+Bad jokes aside, this is a mod making event that is set to take place in December from the 1st till the 31st.  
+Participants in this event are encouraged to create mods that fit a winter, festive or new years theme. Festive can apply to any tradition, event, or religious ceremony that takes place during this time of year.  
```

Comment:
"new years" is superfluous because it is already part of the festive definition. Not 100% against this, but I do think it makes this a bit harder to read.